en/components/transaction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Transaction Service
+_description: 
+_keywords: Ignite UI for Angular, transaction
+---
+
+
+## Transaction Service
+
+The Transaction Service is an injectable middleware (through Angular's DI) that a component may use to accumulate changes without immediately affecting the underlying data.
+    <img src="https://cdn-images-1.medium.com/max/800/1*O-6DidcFW_XCSqgKRfXf_Q.png"
+        style="display:flex;max-height:400px;margin:auto auto 20px auto;" />
```

Comment:
Note that `transform data through pipe` in the image is not a mandatory operation.